Assets/Scripts/Remote/RemoteStatusHandler.cs

```diff
@@ -124,6 +124,12 @@ private void MessageReceived(string message)
                     case "TIME":
                         DayNightManager.Instance.SetTimeOfDay(float.Parse(parameters[1]));
```

Comment:
This breaks ambient sound b/c the events dont trigger, but this is rather a problem of DayNightManager. i'll check if this aplies to the sleep mechanic aswell.